src/test/javascript/spec/component/shared/notification/notification-sidebar.component.spec.ts

```diff
@@ -128,6 +128,18 @@ describe('Notification Sidebar Component', () => {
             expect(notificationService.subscribeToLoadingStateUpdates).toHaveBeenCalledOnce();
             expect(notificationSidebarComponent.sortedNotifications).toHaveLength(notifications.length);
         });
+
+        /**
+         * The notification-sidebar overlaps the exam summary when being printed,
```

Comment:
I don't quite get this test. It just tests that the notification sidebar exists, not that it's hidden. Or am I missing something?

Reply:
Yes exactly, I want to prevent that somebody removes the id as it seems unused because it is not referenced in the css and might seem unnecessary as this would break the PDF export _(again)_

Reply:
I extended the comment for the test, so it is clearer why it exists and what to do if it fails

Reply:
Looks good, thank you!